src/clientStub.ts

```diff
@@ -274,21 +288,21 @@ export class DgraphClientStub {
     }
 
     public async fetchUiKeywords(): Promise<UiKeywords> {
-      return this.callAPI("ui/keywords", {});
+      return this.callAPI("ui/keywords", {...this.options});
```

Comment:
`{...this.options}` is equivalent to `this.options` (maybe there are slight differences I don't know of but latter is fewer characters to type and fewer cpu cycles to process)
I suggest we keep it simple here

There are two more instances below in this file that this comment applies to

Reply:
fixed